### DISCUSSION TITLE
how can I play embedded media in epub on ubuntu 22.04

### DISCUSSION THREAD

**Discussion by zhgxn** [johnfactotum/foliate#944] [General]
**Question:**
I can't play media inside the epub files. when I click the play button, there is no response. 
I want to know what player Foiate use to play these files , which I may not set up properly. 
or do I need install other plugins to make it work ?

**Version:**
 - Foliate version:2.6.4
 - OS/Distribution and version: Ubuntu 22.04
 - Desktop environment: default desktop
 - Installation method: ubuntu ppa

**Comment by zhgxn** [comment]
I extracted the page and opened in epiphany browser, got the same result . So the problem should be related to the webkit engine on my ubuntu . the media file is mp3.

**Comment by johnfactotum** [comment]
> So the problem should be related to the webkit engine on my ubuntu

You might be missing some gstreamer plugins, which WebKitGTK uses for media playback, I think.
